qpid.te

```diff
@@ -29,6 +29,7 @@ files_pid_file(qpidd_var_run_t)
 # Local policy
 #
 
+allow qpidd_t qpidd_var_lib_t:file map;
```

Comment:
Allow rule is correct, this will fix #BZ1519948 , but could you please add this rule to line 51 in this file? We're trying to have similar rules near. And this rule is connected to accessing qpidd_var_lib_t labels. 

Thanks. 
Lukas.